tests/ci_build/ci_build.sh

```diff
@@ -178,6 +178,7 @@ ${DOCKER_BINARY} run --rm --pid=host \
     -e "CI_BUILD_GID=$(id -g)" \
     -e "CUDA_ARCH=-gencode arch=compute_52,code=[sm_52,compute_52] --fatbin-options -compress-all" \
     -e "MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0" \
+    -e "ARCH_OPT=-mavx2" \
```

Comment:
Please make this an environment var of the build_cuda dockerfile as otherwise, this will break the build on ARM-based devices